src/foam/nanos/logger/ILogger.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.logger;
+
+public interface ILogger {
```

Comment:
But then to be consistent we would need to prefix all of our other interfaces with "I" as well, which we don't do. Better to name it NanoLogger or just name it Logger, and then refer to the java Logger with the full pathname in our code where we need to use it.